src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Random random = new Random();
+    private final Color[] colors = Color.values();
+    private final int maxIndex = colors.length;
```

Comment:
I think these vars are too method-specific to be class variables, so let's move them

Reply:
I would argue about how specific these variables are because there would be excessive  method/field calls every time 
 the getRandomColor() method is being called.